hw2/02-search.js

```diff
@@ -1,3 +1,64 @@
-// Add your code here
 
-userInput.addEventListener('click', handleClick);
+function handleClick()
+{
+	console.log('Button clicked!');
+	const txt = document.getElementById('userInput');
+	console.log(txt.value);
+	
+	const items = document.querySelectorAll('.card');
+	for (let i = 0; i < items.length; i++) {
+        const card = items[i];
+		var title = card.querySelector('.card-title')
+		if(title.innerHTML.includes(txt.value))
+		{
+			console.log("-"+txt.value+"-");
+			title.innerHTML = title.innerHTML.replace(txt.value,"<mark>"+txt.value+"</mark>")
+			console.log(title.textContent);
+			card.classList.remove("d-none");
+		}
+	}
+}
+
+
+
+// Function to generate HTML cards based on peopleData
+function generateCards() {
+    const cardContainer = document.getElementById("card-container");
+
+    peopleData.forEach(person => {
+		
+        const card = document.createElement("div");
+        card.classList.add("card");
+        card.classList.add("m-2");
+        card.classList.add("flex-item");
+		card.classList.add("d-none");
+        card.style.width = "12rem";
+
+        // Card content (name and birth year)
+        const cardBody = document.createElement("div");
+        cardBody.classList.add("card-body");
+
+        const cardTitle = document.createElement("h5");
+        cardTitle.classList.add("card-title");
+        cardTitle.innerHTML = person.name;
```

Comment:
WAVE extension flagged this as an alert about a heading jump (from h1 for the exercise title straight to h5)